lib/core/presentation/pages/create_post/create_post_page.dart

```diff
@@ -50,34 +53,49 @@ class CreatePostPage extends StatelessWidget {
+            Divider(color: colorScheme.onSurface),
             Expanded(
               child: Padding(
                 padding: EdgeInsets.symmetric(horizontal: Spacing.smMedium),
                 child: TextFormField(
                   controller: textCtrl,
                   maxLines: 10,
-                  cursorColor: LemonColor.white,
+                  cursorColor: colorScheme.onPrimary,
                   decoration: InputDecoration.collapsed(hintText: t.home.whatOnYourMind),
                 ),
               ),
             ),
-            Divider(color: LemonColor.white.withOpacity(0.7)),
+            Divider(color: colorScheme.onSurface),
             Padding(
               padding: EdgeInsets.symmetric(horizontal: Spacing.smMedium),
               child: Row(
                 children: [
-                  Assets.icons.icHouseParty.svg(color: LemonColor.white.withOpacity(0.7)),
-                  SizedBox(width: Spacing.medium),
-                  Assets.icons.icCrystal.svg(color: LemonColor.white.withOpacity(0.7)),
-                  SizedBox(width: Spacing.medium),
-                  Assets.icons.icTicket.svg(color: LemonColor.white.withOpacity(0.7)),
-                  SizedBox(width: Spacing.medium),
-                  Assets.icons.icPoll.svg(color: LemonColor.white.withOpacity(0.7)),
-                  SizedBox(width: Spacing.medium),
-                  Assets.icons.icCamera.svg(color: LemonColor.white.withOpacity(0.7)),
-                  SizedBox(width: Spacing.medium),
                   Expanded(
+                    flex: 7,
+                    child: Row(
+                      children: [
+                        ThemeSvgIcon(
+                          color: colorScheme.onSurface,
+                          builder: (filter) => Assets.icons.icHouseParty.svg(colorFilter: filter),
+                        ),
+                        SizedBox(width: Spacing.medium),
+
+                        ///Temporary remove since we don't have integration with it yet
+                        // Assets.icons.icCrystal.svg(color: LemonColor.white.withOpacity(0.7)),
+                        // SizedBox(width: Spacing.medium),
+                        // Assets.icons.icTicket.svg(color: LemonColor.white.withOpacity(0.7)),
+                        // SizedBox(width: Spacing.medium),
+                        // Assets.icons.icPoll.svg(color: LemonColor.white.withOpacity(0.7)),
+                        // SizedBox(width: Spacing.medium),
+                        ThemeSvgIcon(
+                          color: colorScheme.onSurface,
+                          builder: (filter) => Assets.icons.icCamera.svg(colorFilter: filter),
+                        ),
+                      ],
+                    ),
+                  ),
+                  Expanded(
+                    flex: 3,
                     child: Container(
                       decoration: BoxDecoration(
                         color: LemonColor.black,
```

Comment:
Please use LemonOutlineButton, and don't set fixed color, may affect theming in the future.